chia/wallet/vc_wallet/cr_cat_drivers.py

```diff
@@ -0,0 +1,591 @@
+from __future__ import annotations
```

Comment:
I thought CR-CATs are coming in a separate PR?

Reply:
In the PR description I explained that the CR-CAT Chialisp is coming in at the same time as the VC Chialisp because they are an important case to test for VCs.